smoketest/graphqloncallassignments_test.go

```diff
@@ -29,33 +29,11 @@ func TestGraphQLOnCallAssignments(t *testing.T) {
 	sql := `insert into escalation_policies (id, name) 
 					values ({{uuid "eid"}}, 'esc policy');`
 
-	h := harness.NewHarness(t, sql, "escalation-policy-step-reorder")
-	defer h.Close()
-
 	type onCallAssertion struct {
 		Service, EP, EPName, User string
 		StepNumber                int
 	}
 
-	doQL := func(t *testing.T, query string, res interface{}) {
-		g := h.GraphQLQueryT(t, query, "/api/graphql")
-		for _, err := range g.Errors {
-			t.Error("GraphQL Error:", err.Message)
-		}
-		if len(g.Errors) > 0 {
-			t.Fatal("errors returned from GraphQL")
-		}
-		t.Log("Response:", string(g.Data))
-
-		if res == nil {
-			return
-		}
-		err := json.Unmarshal(g.Data, &res)
-		if err != nil {
-			t.Fatal(err)
-		}
-	}
-
 	var idCounter int
```

Comment:
Since this is used for names per-harness, it can/should live within `t.Run`. Otherwise we need to use something like `atomic.AddInt32` to handle the concurrent access.